test/unit/proxy_test.rb

```diff
@@ -192,17 +192,10 @@ def test_deployable
   end
 
   def test_apicast_configuration_driven
-    @proxy.provider.stubs(:provider_can_use?).with(:apicast_v1).returns(true)
     @proxy.provider.stubs(:provider_can_use?).with(:apicast_v2).returns(true)
 
     @proxy.apicast_configuration_driven = true
     assert @proxy.apicast_configuration_driven
-
-    @proxy.apicast_configuration_driven = false
-    refute @proxy.apicast_configuration_driven
```

Comment:
Do we remove the whole field? If not, why this test should be removed?

Reply:
apicast_configuration_driven method is only checking if the provider can use apicast_v2 so I don't think this test is need

Reply:
@HemaHG So do you think we should keep this test then? and Why?